packages/styleguide/stories/Molecules/Alert/Alert.stories.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import Alert from '../../gamut-templates/src/Alert';
+import { LayoutGrid, Column } from '@codecademy/gamut/src/Layout';
+
+export default {
+  component: Alert,
+  title: 'Templates/Alert',
+};
```

Comment:
P.S. you might enjoy making it use this:

```suggestion
export default decoratedStory("Molecules", Alert);
```